inginious/frontend/user_manager.py

```diff
@@ -651,7 +651,33 @@ def user_saw_task(self, username, courseid, taskid):
             upsert=True
         )
 
-    def update_user_stats(self, username, course, task, submission, result_str, grade, state, newsub, task_dispenser):
```

Comment:
This signature is changed but I don't see any change in its usage. Is this a fix ?

Reply:
OK but then you need to update its usage : https://github.com/AlexandreDoneux/INGInious/blob/UI_course_list/inginious/frontend/submission_manager.py#L79